core/include/jiminy/core/engine/EngineMultiRobot.h

```diff
@@ -691,6 +691,24 @@ namespace jiminy
             Eigen::MatrixBase<TangentVectorType2>                  const & tau,
             pinocchio::container::aligned_vector<ForceDerived>     const & fext);
 
+        /// \brief Compute system acceleration from current system state.
+        ///
+        /// \details This function performs forward dynamics computation, either
+        ///          with kinematic constraints (using Lagrange multiplier for computing the forces)
+        ///          or unconstrainted (aba).
+        ///
+        /// \param[in] robot Robot for which to compute the dynamics.
+        /// \param[in] q Joint position.
+        /// \param[in] v Joint velocity.
+        /// \param[in] u Joint torque.
+        /// \param[in] fext External forces applied on the system.
+        /// \return System acceleration.
+        vectorN_t computeAcceleration(std::shared_ptr<Robot> robot,
```

Comment:
Use `systemDataHolder_t  & system` instead of `std::shared_ptr<Robot>` to have a more consistent API.